app/javascript/react/components/ReviewForm.js

```diff
@@ -14,9 +14,20 @@ const ReviewForm = (props) => {
     })
   }
 
+  const clearForm = () => {
+    setNewReview({
+      rating: 0,
+      body: "",
+      speed_data: 0
```

Comment:
you set speed_data to 0 here, but the initial speed_data is an empty string, lets make these consistent